packages/orbit-design-tokens/src/__tests__/index.test.ts

```diff
@@ -43,40 +43,32 @@ describe("getTokens should accept some palette and base foundation", () => {
   });
   it("should deep merge", () => {
     expect(theme.paletteProductLight).toBe(foundation.palette.product.light);
-    expect(theme.fontSizeTextNormal).toBe(foundation.base.fontSizeMd);
+    expect(theme.fontSizeTextNormal).toBe(foundation.fontSize.normal);
   });
 });
 
 describe("getTokens should accept some base", () => {
   const brand = {
-    base: {
-      iconSizeSm: "16px",
-      iconSizeMd: "20px",
-      iconSizeLg: "24px",
-      sizeSm: "1px",
-      sizeMd: "2px",
-      sizeLg: "3px",
-      sizeXl: "4px",
-      size2xl: "5px",
+    size: {
+      small: "1px",
+      medium: "2px",
+      large: "3px",
+      extraLarge: "4px",
+      extraExtraLarge: "5px",
     },
   };
+
   const theme = getTokens(brand);
   it("should match snapshot", () => {
     expect(theme).toMatchSnapshot();
   });
   it("tokens should have those colors", () => {
-    expect(theme.widthIconSmall).toBe(brand.base.iconSizeSm);
-    expect(theme.heightIconSmall).toBe(brand.base.iconSizeSm);
-    expect(theme.widthIconMedium).toBe(brand.base.iconSizeMd);
-    expect(theme.heightIconMedium).toBe(brand.base.iconSizeMd);
-    expect(theme.widthIconLarge).toBe(brand.base.iconSizeLg);
-    expect(theme.heightIconLarge).toBe(brand.base.iconSizeLg);
-    expect(theme.heightInputNormal).toBe(brand.base.sizeXl);
-    expect(theme.heightInputLarge).toBe(brand.base.size2xl);
-    expect(theme.heightInputSmall).toBe(brand.base.sizeLg);
-    expect(theme.heightButtonNormal).toBe(brand.base.sizeXl);
-    expect(theme.heightButtonLarge).toBe(brand.base.size2xl);
-    expect(theme.heightButtonSmall).toBe(brand.base.sizeLg);
+    expect(theme.heightInputNormal).toBe(brand.size.extraLarge);
```

Comment:
I meant if we change `heightInputNormal` to use any other token, this test will fail. But what's being tested here won't be broken. This test relies on `heightInputNormal` using one token, but that can change. My suggestion would be to have a custom token (let's call it `customToken`) that uses `foundation.size.extraLarge` and assert that when we customize `extraLarge` the `customToken` also gets the new value, which is exactly what this test is supposed to test. That way, if we ever change `heightInputNormal`, this test won't fail, as it shouldn't

Reply:
I will leave it as it is because otherwise, we need to use another function, not the `getTokens`. In case, if the default tokens that depended on the base will be changed - it's not a problem to adjust the test, as it's expected will be failed.

Reply:
I will leave it as it is, in case it will be changed - it's expected to adjust the test also in that case, not a lot of extra work. Currently, this function is typed and we will need to add suppression in order to add "fake" token